auth-api/migrations/versions/2024_05_07_a8c61064e6fd_add_eft_overdue_suspension_code.py

```diff
@@ -0,0 +1,29 @@
+"""Add Overdue EFT Payments suspension code
+
+Revision ID: a8c61064e6fd
+Revises: e65cc8ab51cc
+Create Date: 2024-05-07 15:53:21.674906
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a8c61064e6fd'
+down_revision = 'e65cc8ab51cc'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("INSERT INTO public.suspension_reason_codes "
+            "(code, description,\"default\") "
+            "VALUES "
+            "('OVERDUE_EFT', 'Overdue EFT Payments', false)")
+    op.execute("commit")
```

Comment:
not sure what this is for, it auto commits?

Reply:
I saw other similar migrations has `op.execute("commit")`, make it consistent
![image](https://github.com/bcgov/sbc-auth/assets/16507223/964bda87-95ed-4152-9ea5-5f75ff6c266b)

Reply:
must have snuck past me no need for `commit`

Reply:
could you fix the migration `2024_04_19_824f5ef280c6` as well? so this doesn't repeat? I ran into the same issue yesterday with `STRR` repeating the same issues

Reply:
ok I'll fix other migrations.